Improve message distribution performance (#61)
tenzir/threatbus: CHANGELOG.md

## Unreleased

- 🎁 The `zmq-app` and `zeek` plugins now use the Unix select system call for
  improved performance during message passing. Both plugins forward messages a

>   lot faster to subscribers than before.

Reviewer: It'd be nice to switch from a qualitative "a lot" statement to a quantitative one.

  [#61](https://github.com/tenzir/threatbus/pull/61)

> - 🎁 Shutdown behavior is now controlled. Pressing ctrl+c first shuts down

Reviewer: ```suggestion
- 🎁 Threat Bus now has a controlled shutdown. Pressing ctrl+c first shuts down
```

  backbone plugins, then app plugins, and lastly Threat Bus itself.
  [#61](https://github.com/tenzir/threatbus/pull/61)

> - ⚠️ A new base class is added to implement plugin-threads. Plugin developers

Reviewer: ```suggestion
- ⚠️ There exists a new base class for implement plugin-threads. Plugin developers
```
